CrawlJob.php

```diff
@@ -32,13 +37,15 @@ class CrawlJob
      * @var string $url
      * @var string $baseUrl
      * @var array  $blacklist
+     * @var array  $whitelist
      * @var array  $metadata
      */
-    public function __construct($url, $baseUrl, array $blacklist = [], array $metadata = [])
+    public function __construct($url, $baseUrl, array $blacklist = [], array $whitelist = [], array $metadata = [])
```

Comment:
For backward compatibility I think it's because to append this argument, also in the admin I use the CrawlJob object.